qemu/tests/cfg/vdi_image_convert.cfg

```diff
@@ -0,0 +1,26 @@
+- vdi_image_convert:
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    required_qemu = [6.1.0-7, )
+    virt_test_type = qemu
+    type = vdi_image_convert
+    start_vm = no
+    create_image = yes
+    remove_image = yes
+    write_size = 512M
+    variants:
+        - to_raw:
+            images = "vdi_src"
+            image_name_vdi_src = "images/vdi_src"
+            image_size_vdi_src = 1G
+            image_format_vdi_src = vdi
+            convert_target = target
+            image_name_target = "images/vdi_to_raw"
+            image_format_target = raw
```

Comment:
a little bit of dup here for src and target, what about:

images = 'src'
image_name_src = 'images/src'
image_size_src = 1G
convert_target = target
image_name_target = "images/target"
variants:
-  to_raw:
   image_format_src = vdi
   image_format_target = raw
-  to_vdi:
   image_format_src = raw
   image_format_target = vdi

Reply:
Yes. Updated, thanks.